pubspec.yaml

```diff
@@ -9,7 +9,7 @@ environment:
 
 dependencies:
   build: '>=0.12.6 <2.0.0'
-  recase: ^2.0.0
+  recase: ^3.0.0
```

Comment:
There is some strict need to use version 3.0.0? If not, then I think that would be best to use a range check: `>= 2.0.0 <3.1.0`, should help to avoid some conflict resolution between other libraries

Reply:
I agree.  thank you for your feedback